concepts/booleans/about.md

```diff
@@ -7,9 +7,9 @@ True || False           -- True
 True && False           -- False
 ```
 
-The three boolean operators each have a different operator precedence. They are evaluated in this order: `not` first, `&&` second, and finally `||`. If you want to bypass these rules, you can enclose a boolean expression in parentheses (`()`), as the parentheses have an even higher operator precedence.
+The three boolean operators each have a different operator precedence. They are evaluated in this order: `not` first, `&&` second, and finally `||`. If you want to on override these rules, you can enclose a boolean expression in parentheses (`()`), as the parentheses have an even higher operator precedence.
```

Comment:
An extra **on** slipped into this by accident, it looks like. It should be `If you want to override these rules`.

Reply:
Edited. Thanks.